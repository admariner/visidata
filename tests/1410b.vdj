#!vd -p
{"sheet": "global", "col": null, "row": "numeric_binning", "longname": "set-option", "input": "True", "keystrokes": "", "comment": null}
{"longname": "open-file", "input": "sample_data/test.jsonl", "keystrokes": "o"}
{"sheet": "test", "col": "key1", "row": "", "longname": "freq-col", "input": "", "keystrokes": "Shift+F", "comment": "open Frequency Table grouped on current column, with aggregations of other columns"}
{"sheet": "test_key1_freq", "col": "key1", "row": "", "longname": "sort-asc", "input": "", "keystrokes": "[", "comment": "sort ascending by current column; replace any existing sort criteria"}
