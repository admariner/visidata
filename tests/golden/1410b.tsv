key1	count	percent	histogram
#ERR	1	10.00	■■■■■■■■■■■■■■■■■■■
	2	20.00	■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■
2016-01-01 11:00:00	1	10.00	■■■■■■■■■■■■■■■■■■■
2016-01-01 1:00	1	10.00	■■■■■■■■■■■■■■■■■■■
2017-12-25 8:44	1	10.00	■■■■■■■■■■■■■■■■■■■
2018-07-27 16:44	1	10.00	■■■■■■■■■■■■■■■■■■■
2018-07-27 18:44	1	10.00	■■■■■■■■■■■■■■■■■■■
2018-07-27 4:44	1	10.00	■■■■■■■■■■■■■■■■■■■
2018-10-20 18:44	1	10.00	■■■■■■■■■■■■■■■■■■■
